server/routes/auth.js

```diff
@@ -26,7 +26,7 @@ router.post("/login", validationMiddleware, async function (req, res, next) {
             return;
         }
         // create and return jwt with user obj
```

Comment:
If the user is a chef do we want to go ahead and that info now? I don't think we have to however if we decide to direct the user to their profile page after login we would need it correct?

Reply:
Yes, we'll have to fetch the user's chef profile info if they are chef, and we can do that via `GET /chefs/`. If the team thinks it'll be easier if we return the entire chef profile with each user response, I'll change the API to do it.

Thoughts ? @sagar-13 @ThinkTink @sina-jamshidi

Reply:
I think it makes sense to return all information about a user here, including the chef details, rather than making two different API calls. You can do if just like Megan did in the[ /user route](https://github.com/hatchways/team-corn-pops/blob/8fbd190420d066442dba7e881cc062a0b824e860/server/routes/auth.js#L100)

Reply:
Just fixed it!